internal/testutil/acc/advanced_cluster.go

```diff
@@ -101,3 +103,29 @@ func CheckExistsClusterHandlingRetry(projectID, clusterName string) error {
 		return nil
 	})
 }
+
+func ConvertToTPFAttrsMap(attrsMap map[string]string) {
+	if !config.AdvancedClusterV2Schema() {
+		return
+	}
+	names := make([]string, 0, len(attrsMap))
+	for name := range attrsMap {
+		names = append(names, name)
+	}
+	for _, name := range names {
+		newName := name
```

Comment:
Maybe extract these 3 lines as a function. Then we can re-use it in the `prefix` for plural data sources and also in `attrSet` checks